totest-manager.py

```diff
@@ -575,6 +582,35 @@ def totest(self):
             self.update_totest(new_snapshot)
             self.write_version_to_dashboard("totest", new_snapshot)
 
+    def send_amqp_event(self, current_snapshot, current_result):
+        if not self.amqp_url:
+            logger.info('No ttm_amqp_url configured in oscrc - skipping amqp event emission')
+            return
+
+        logger.info('Sending AMQP message')
+        msg_topic = '%s.ttm.build.%s' % (self.project_base.lower(), self._result2str(current_result))
+        msg_body = json.dumps({
+            'build': current_snapshot,
+            'project': self.project,
+            'failed_jobs': self.failed_jobs,
+            'ignored_failed_jobs': self.ignored_failed_jobs,
+        })
```

Comment:
Can you give us an example message in the PR description?

Reply:
done